packages/reference/src/common/SortableLinkList.tsx

```diff
@@ -1,23 +1,27 @@
-import * as React from 'react';
-import {
-  SortableContainer,
-  SortableContainerProps,
-  SortableElement,
-  SortableHandle,
-} from 'react-sortable-hoc';
+import React from 'react';
 
 import tokens from '@contentful/f36-tokens';
+import { DndContext, DragEndEvent, DragStartEvent } from '@dnd-kit/core';
+import { SortableContext, SortingStrategy, arrayMove, useSortable } from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
 import { css, cx } from 'emotion';
 
+import { useSortIDs } from '../utils/useSortIds';
 import { ReferenceEditorProps } from './ReferenceEditor';
 
 const styles = {
   container: css({
     position: 'relative',
   }),
   item: css({
+    position: 'relative',
     marginBottom: tokens.spacingM,
-    zIndex: tokens.zIndexModal, // setting this to an index above 99 fixes dragged item disappearing issue. Should not be higher than 100 so it does not overlap the asset modal.
```

Comment:
should we keep this comment? can maybe prevent bugs in the future :)

Reply:
Not sure how it was working previously or maybe it was, but with react-sortable-hoc. It hasn't had `position: relative;`. Currently I'm setting it only on drag.